RELEASE-NOTES.md

```diff
@@ -22,6 +22,8 @@
 ## New additions
 * Added support for `title` field in Streamlit definition in `snowflake.yml` project file.
 * Added `--auto-compress` flag to `snow stage copy` command enabling use of gzip to compress files during upload.
+* Added connection option `--token-file-path` allowing passing OAuth token using a file. The function is also
+  supported by setting `token_file_path` in connection definition.
```

Comment:
Do we want to permission checks like config.toml has?

Reply:
Why should we? What scenario we see as risky? If the input token is invalid then Snowflake will not authorize the connection.

Reply:
You are right.